compiler/rustc_ast_lowering/src/path.rs

```diff
@@ -155,7 +155,7 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
         }
 
         // Simple case, either no projections, or only fully-qualified.
-        // E.g., `std::mem::size_of` or `<I as Iterator>::Item`.
```

Comment:
this is just a random example of a multi segment path, so please choose another one. maybe std::iter::zip or something?

Reply:
I just reverted that